README.md

```diff
@@ -4,12 +4,19 @@ Streams /r/neoliberal's busyass discussion thread.
 
 [Now available on the Google Play Store](https://play.google.com/store/apps/details?id=ca.inhumantsar.tacostream)
 
-## Build
+## Build instructions
+This project requires a Flutter version below 3.0.0 and above 2.7.0, meaning one must downgrade their Flutter version to 2.10.5 in order to build the project.
 
-Just like any other Flutter project.
+Downgrade Flutter by navigating to your SDK path:
 
-        flutter build
+        cd /path/to/flutter/sdk
```

Comment:
i think you'll need to add a newline after this for the code fencing to work properly